docpages/example_code/editing_messages.cpp

```diff
@@ -46,9 +43,37 @@ int main() {
 				/* change the channel name and edit the channel itself */
 				channel.set_name(name);
 				bot.channel_edit(channel);
-
 				event.reply("Channel name is now `" + name + "`.");
 			});
+		} else if (event.command.get_command_name() == "embed-send") {
+			dpp::embed embed = dpp::embed()
+				.set_color(dpp::colors::sti_blue)
+				.set_title("like and subscribe")
+				.set_url("https://dpp.dev/")
+				.set_author("Some author", "https://dpp.dev/", "https://dpp.dev/DPP-Logo.png")
+				.set_description("Creator is <creator name>");
+			event.reply(embed);
+		} else if (event.command.get_command_name() == "embed-edit") {
+			const auto description = std::get<std::string>(event.get_parameter("desc"));
+
+			/* get the message to edit its embed after. here string will automatically be converted to snowflake */
+			const dpp::snowflake msg_id = std::get<std::string>(event.get_parameter("msg-id"));
+
+			bot.message_get(msg_id, event.command.channel_id, [&bot, description, event](const dpp::confirmation_callback_t& callback) {
+				if (callback.is_error()) {
+					event.reply("error");
+					return;
+				}
+				auto message = callback.get<dpp::message>();
+				auto& embeds = message.embeds;
+
+				/* change the embed description and edit the message itself.
```

Comment:
This doesn't follow our Code Conventions.

Multi-line comments should be made in the following format:
```cpp
/* thing
 * thing two
 * thing three
 */
```